src/components/Filters/components/LocationsFilter.js

```diff
@@ -60,7 +60,7 @@ const buildNode = (id, count, t) => ({
 
 const LocationsFilter = props => {
   const cIDs           = useSelector(state => selectors.getEventsByType(state.events)[CT_CONGRESS]);
-  const congressEvents = useSelector(state => (cIDs || []).map(x => mdb.getDenormCollection(state.mdb, x)));
```

Comment:
Here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#return_value written: A new array with the elements that pass the test. If no elements pass the test, an empty array will be returned. So you shouldn't use the last `|| []`

Reply:
fixed